feat(ensanalytics): Apply Enhanced Program Rules (#1307)
namehash/ensnode: .changeset/sharp-mammals-carry.md

---
"ensapi": patch
---

> Use `slowestChainIndexingCursor` from indexing status as the aggregated referrer metrics cache `updatedAt` timestamp instead of current system time. This ensures the timestamp accurately reflects the indexer state rather than the current system time, preventing issues with clock skew between systems.

Reviewer: ```suggestion
Use `slowestChainIndexingCursor` from indexing status as the referrer leaderboard cache `updatedAt` timestamp instead of current system time. This optimizes the timestamp to reflect the indexer state rather than the current system time.
```
